source-kinesis/capture.go

```diff
@@ -55,8 +58,9 @@ func readStream(ctx context.Context, shardRange airbyte.Range, client *kinesis.K
 type streamReader struct {
 	client             *kinesis.Kinesis
 	ctx                context.Context
+	idx                int
```

Comment:
I _think_ this is the binding index, right? It might be more clear to call this 'bindingIndex' or something which makes the purpose clearer to the reader.